container-chains/runtime-templates/frontier/src/migrations.rs

```diff
@@ -77,6 +77,53 @@ where
     }
 }
 
+pub struct MigratePrecompileProxyDummyCode<T>(pub PhantomData<T>);
```

Comment:
Maybe in the future we can have a generic migration in moonkit that we pass it the precompile address and it automatically writes the bytecode?

Reply:
That's a neat idea. For sure let's do that for the next one